registrations/tasks.py

```diff
@@ -708,13 +708,14 @@ def is_registered_on_whatsapp(self, address):
         Returns whether or not the number is recognised on wassup
         """
         r = requests.post(
-            settings.WASSUP_URL,
-            json={"number": settings.WASSUP_NUMBER, "msisdns": [address], "wait": True},
-            headers={"Authorization": "Token {}".format(settings.WASSUP_TOKEN)},
+            settings.ENGAGE_URL,
```

Comment:
I think we should move away from specifying the exact URL endpoint for the contact check (eg `whatsapp.praekelt.org/v1/contacts`), and move towards specifying the base url (eg `whatsapp.praekelt.org`), and then using `urljoin` (https://docs.python.org/3.5/library/urllib.parse.html#urllib.parse.urljoin) to get the specific path.

Right now, because this is the only endpoint, it's not necessary, but I think we should plan for the future for when we want to access different endpoints on the engage API.

Reply:
so I should not use `settings.ENGAGE_URL`?

Reply:
I think we should be using `settings.ENGAGE_URL`, but we should assume that `ENGAGE_URL` is something like `whatsapp.praekelt.org`, not the full path of `whatsapp.praekelt.org/v1/contacts`, and that we need to add `v1/contacts` to `ENGAGE_URL` to be able to get the full URL that we need to call for the contact check.